server/src/app.ts

```diff
@@ -1,13 +1,22 @@
+import cors from "cors";
 import express from "express";
 import { withDb, withEnv } from "./lib";
 import { withMorgan } from "./lib/middleware/with-morgan";
 import { healthRouter } from "./routes";
 import { studentsRouter } from "./routes";
 
+// TODO(amantri): Get client port from env. Do we add this only on dev env?
```

Comment:
I try not to use wildcards for issues like this in case we forget to remove it before deploying :)

It's better to have things break during development than to introduce security issues.